triage/templates/triage/custom-page.html

```diff
@@ -40,6 +41,102 @@ <h2 class="heading-large">Connect with international buyers</h2>
   </div>
 </section>
 {% endif %}
+{% if persona.name == 'OCCASIONAL_EXPORTER'%}
+<section>
+  <div class="container">
+    <div class="grid-row">
+      <div class="column-half">
+        <h2 class="heading-large">Explore new markets</h2>
+        <p>Learn about the best markets for your sector, increase your exporting knowledge and start connecting with overseas buyers.</p>
+        {% if not sso_is_logged_in %}
+          <p><a class="link" href="{{ sso_register_url }}">Register</a> or <a class="link" href="{{ sso_login_url }}">sign in</a> to save your progress.</p>
+        {% endif %}
+      </div>
+    </div>
+  </div>
+</section>
+{% endif %}
+{% if triage_result.is_exporting_goods %}
+<section id="sector">
+  <div class="container">
+    <div class="grid-row">
+      <div class="column-half">
+        <form action="/custom/" method="post">
+          <h2 class="heading-large">Select the goods you'd like to export</h2>
+            {% if triage_result.sector and not service_sector %}
+            <p>Select a different product category.</p>
+            {% else %}
+            <p>Type the product you want to export to get statistics on the largest importers.</p>
+            {% endif %}
+            <div class="form-group{% if service_sector %} form-group-error{% endif %}">
+              {% if service_sector %}
+              <p>No data available, please select a different commodity.</p>
+              {% endif %}
+                {% csrf_token %}
```

Comment:
just a note on convention: csrf_token is normally the first thing just after `<form>`, so 

```
<form>
{% csrf_token %}
...
```

as it's a convention lazy devs reading it may thing `csrf_token` is not present as it's not the first thing after `<form>`